.agent/scratch/archive/005-thorough-vetting-campaign.md

```
@@ -0,0 +1,268 @@
+  - adopted the reduced read profile in the product code (`32 MiB` cache, `128 MiB` mmap)
+- Live command measurements after both fixes:
+  - `polylogue --provider codex --latest list --format json`
+    - before: `14.22s`, `2012944 KB`
+    - after summary projection only: `3.27s`, `813396 KB`
+    - after read tuning: `3.33s`, `117712 KB`
+  - `polylogue --provider codex --latest open --print-path`
+    - before: `14.15s`, `2014496 KB`
+    - after summary projection only: `3.21s`, `813100 KB`
+    - after read tuning: `3.20s`, `117528 KB`
+
+## 2026-04-11 artifact-proof fix
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)